plugins/services/services_inspector.rb

```diff
@@ -31,7 +31,7 @@ def inspect(_filter, _options = {})
       when @system.has_command?("systemctl")
         then [inspect_systemd_services, "systemd"]
       when @system.has_command?("initctl") && @system.has_command?("chkconfig")
-        then [parse_redhat_chkconfig, "upstart"]
+        then [parse_redhat_chkconfig.map{ |s| s["legacy_sysv"] = true; s }, "upstart"]
```

Comment:
Do not use semicolons to terminate expressions.<br>Space missing to the left of {.